src/backgroundblurprocessor/BackgroundBlurVideoFrameProcessor.ts

```diff
@@ -62,11 +62,17 @@ export default class BackgroundBlurVideoFrameProcessor extends BackgroundFilterV
     spec?: BackgroundFilterSpec,
     options?: BackgroundBlurOptions
   ): Promise<BackgroundBlurProcessor | undefined> {
-    spec = BackgroundBlurVideoFrameProcessor.resolveSpec(spec);
-    options = BackgroundBlurVideoFrameProcessor.resolveOptions(options);
-    const { logger } = options;
+    let processorSpecs: BackgroundFilterSpec = { ...spec };
+    let processorOptions: BackgroundBlurOptions = { ...options };
```

Comment:
I would move this into `resolveSpec` and `resolveOptions` and leave the previous code alone.

Reply:
done